packages/RichTextEditing-Core.package/RichTextHTMLExportStrategy.class/instance/handleEndOfStructure..st

```diff
@@ -0,0 +1,6 @@
+stop-condition-handling
+handleEndOfStructure: anEvent
+
+	self putClosingStrings.
+	self removeStructuresFromActive: anEvent second.
+	self putOpeningStrings
```

Comment:
Why putting opening strings after end of one structure? Shouldn't that only be called on beginning of structure?

Reply:
If text structures overlap, you will have to close all tags before opening the remaining ones again, because otherwise you will not produce valid HTML

Reply:
Imo this should be commented then, as it seems counter intuitive, that an end of a structure implies opening the new ones